pkg/k8shandler/forwarding.go

```diff
@@ -52,27 +53,13 @@ func (clusterRequest *ClusterLoggingRequest) generateCollectorConfig() (config s
 		}
 	}
 
-	generator, err := forwarding.NewConfigGenerator(
-		clusterRequest.Cluster.Spec.Collection.Logs.Type,
-		clusterRequest.includeLegacyForwardConfig(),
-		clusterRequest.includeLegacySyslogConfig(),
-		clusterRequest.useOldRemoteSyslogPlugin(),
-	)
-
-	if err != nil {
```

Comment:
Was this moved someplace?  We lost validation that returns status that there is no collector